### DISCUSSION TITLE
how to read [dot]env from custom module

### DISCUSSION THREAD

**Discussion by iqraaaswad** [directus/directus#8164] [Archive]
i wanna put my variable at .env.. How to read that variable from custom module ?

**Comment by joselcvarela** [comment]
Hello,
If you are talking about this https://docs.directus.io/guides/modules/ currently it's not supported.
Environment variables are only available at server side at the moment.

**Reply by rijkvanzanten** [reply]
> Environment variables are only available at server side at the moment.

Which won't change either, as those environment variables contain private information that you wouldn't want to leak to the front-end.

**Reply by iqraaaswad** [reply]
@rijkvanzanten ah i see.. thanks for your explanation
